server/src/main/java/org/opensearch/extensions/action/ExtensionsAction.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.action.ActionType;
+
+public class ExtensionsAction extends ActionType<ExtensionsActionResponse> {
+    public static final String NAME = "cluster:monitor/extension";
```

Comment:
Should be keep the action name in the same format as `internal...`?

Reply:
Sure makes sense.